<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - Test with firefox</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>Test with firefox</h1>
<p>
  Test duration: 135.690s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">testcases.newsfeed.PostFunction.NormalPost.TestcaseOfPost</td>
      </tr>
            
<tr>
  <td class="method">
                  TC03_Check_Button_Shared_Post
      </td>
  <td class="duration">
    42.038s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.Throwable: Mulitple failures (2):<br/>
Failure 1 of 2<br/>
java.lang.AssertionError: expected [true] but found [false]<br/>
	at org.testng.Assert.fail(Assert.java:94)<br/>
	at org.testng.Assert.failNotEquals(Assert.java:494)<br/>
	at org.testng.Assert.assertTrue(Assert.java:42)<br/>
	at org.testng.Assert.assertTrue(Assert.java:52)<br/>
	at CommonHelper.Function.AbstractTest.verifyTrue(AbstractTest.java:67)<br/>
	at testcases.newsfeed.PostFunction.NormalPost.TestcaseOfPost.TC03_Check_Button_Shared_Post(TestcaseOfPost.java:111)<br/>
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)<br/>
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br/>
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br/>
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br/>
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br/>
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br/>
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br/>
	at org.testng.TestRunner.privateRun(TestRunner.java:767)<br/>
	at org.testng.TestRunner.run(TestRunner.java:617)<br/>
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br/>
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br/>
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br/>
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)<br/>
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br/>
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br/>
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br/>
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br/>
	at org.testng.TestNG.run(TestNG.java:1057)<br/>
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)<br/>
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)<br/>
<br/>
Failure 2 of 2<br/>
java.lang.AssertionError: expected [false] but found [true]</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.Assert.fail(Assert.java:94)<br />
              org.testng.Assert.failNotEquals(Assert.java:494)<br />
              org.testng.Assert.assertFalse(Assert.java:63)<br />
              org.testng.Assert.assertFalse(Assert.java:73)<br />
              CommonHelper.Function.AbstractTest.verifyFalse(AbstractTest.java:78)<br />
              testcases.newsfeed.PostFunction.NormalPost.TestcaseOfPost.TC03_Check_Button_Shared_Post(TestcaseOfPost.java:122)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.base/java.lang.reflect.Method.invoke(Method.java:564)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:767)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:240)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br />
              org.testng.TestNG.run(TestNG.java:1057)<br />
              com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)<br />
              com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  TC04_VALIDATE_POST_CONTENT
      </td>
  <td class="duration">
    77.976s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.NullPointerException</b></a><br />
    <div class="stackTrace" id="exception-1">
              java.base/java.util.Objects.requireNonNull(Objects.java:222)<br />
              org.openqa.selenium.support.ui.FluentWait.&lt;init&gt;(FluentWait.java:106)<br />
              org.openqa.selenium.support.ui.WebDriverWait.&lt;init&gt;(WebDriverWait.java:85)<br />
              org.openqa.selenium.support.ui.WebDriverWait.&lt;init&gt;(WebDriverWait.java:45)<br />
              CommonHelper.Function.AbstractPage.waitElementToVisible(AbstractPage.java:371)<br />
              Newsfeed.Editor.Common.Function.clickToCreatePost(Function.java:120)<br />
              testcases.newsfeed.PostFunction.NormalPost.TestcaseOfPost.TC04_VALIDATE_POST_CONTENT(TestcaseOfPost.java:142)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.base/java.lang.reflect.Method.invoke(Method.java:564)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:767)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:240)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br />
              org.testng.TestNG.run(TestNG.java:1057)<br />
              com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)<br />
              com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)<br />
                </div>
      </td>
</tr>
      </table>


  <table class="resultsTable">
    <tr><th colspan="3" class="header passed">Passed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">testcases.newsfeed.PostFunction.NormalPost.TestcaseOfPost</td>
      </tr>
            
<tr>
  <td class="method">
                  TC01_Check_Display_of_Editor_Case_New_Post
      </td>
  <td class="duration">
    0.355s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
      </table>
  
</body>
</html>
